src/containerapp/azext_containerapp/_params.py

```diff
@@ -440,3 +440,36 @@ def load_arguments(self, _):
         c.argument('workload_profile_type', help="The type of workload profile to update. Run 'az containerapp env workload-profile list-supported -l <region>' to check the options for your region.")
         c.argument('min_nodes', help="The minimum node count for the workload profile")
         c.argument('max_nodes', help="The maximum node count for the workload profile")
+
+    # Container App job
+    with self.argument_context('containerapp job') as c:
+        c.argument('name', name_type, metavar='NAME', id_part='name', help=f"The name of the Container Apps Job. A name must consist of lower case alphanumeric characters or '-', start with a letter, end with an alphanumeric character, cannot have '--', and must be less than {MAXIMUM_CONTAINER_APP_NAME_LENGTH} characters.")
+        c.argument('cron_expression', help='Cron expression. Only supported for trigger type "Schedule"')
+        c.argument('image', help="Container image, e.g. publisher/image-name:tag.")
+        c.argument('replica_completion_count', options_list=['--replica-completion-count', '--rcc'], help='Number of replicas that need to complete successfully for execution to succeed.')
+        c.argument('replica_retry_limit', help='Maximum number of retries before the replica fails.')
+        c.argument('replica_timeout', help='Maximum number of seconds a replica can execute.')
+        c.argument('parallelism', help='Maximum number of replicas to run per execution.')
+        c.argument('workload_profile_name', options_list=['--workload-profile-name', '-w'], help='The friendly name for the workload profile')
+        c.argument('min_executions', type=int, help="Minimum number of job executions that are created for a trigger, default 0.")
+        c.argument('max_executions', type=int, help="Maximum number of job executions that are created for a trigger, default 100.")
+        c.argument('polling_interval', type=int, help="Interval to check each event source in seconds. Defaults to 30s.")
+
+    with self.argument_context('containerapp job create') as c:
+        c.argument('system_assigned', help='System assigned identity.')
```

Comment:
For system assigned identity, if the expected behavior is when --system-assigned is specified, then we will use system identity, you could add `action='store_true'`

Reply:
If add 'is_preview=True' to command group, required to add `action='store_true'` for arguments which default value is 'true' .
Otherwise the performance will be inconsistent with not adding the flag 'is_preview=True'